amgut/db/patches/0017.sql

```diff
@@ -0,0 +1,195 @@
+-- August 17, 2015
+-- Rearrange barcodes to be attached to the correct projects
+Do $do$
+DECLARE
+    ag_pid bigint;
+    pid bigint;
+BEGIN
+ag_pid := (SELECT project_id FROM barcodes.project WHERE project = 'American Gut Project');
+
+-- Assign british gut kits to british gut and american gut
+CREATE TEMP TABLE hold_table (
+    barcode varchar,
+    project_id bigint
+);
+
+pid := (SELECT project_id FROM barcodes.project WHERE project = 'British Gut Project');
```

Comment:
`ag_pid` is constant over everything, since it is the american gut project id, so having it queried once and stored makes sense. `pid` is changing for each project that is being updated. Because you need to declare your variables at the time of function creation, it's easier to declare the two variables and reuse `pid` for each project.